tests/storage/backends/test_inmem.py

```diff
@@ -28,7 +29,7 @@ def test_can_schedule_single_job(self, defaultbackend, simplejob):
         new_job = defaultbackend.get_job(job_id)
 
         # Does the returned job record the function we set to run?
-        assert str(new_job.func) == str(id)
+        assert str(new_job.func) == stringify_func(id)
```

Comment:
I'm pretty sure this will fire an error once #8 is merged and you install the precommit hooks.